packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.tsx

```diff
@@ -0,0 +1,159 @@
+            <div
+                className='compare-cfd-header-navigation'
+                onClick={() => {
+                    history.push(routes.traders_hub);
+                }}
+            >
+                <Icon icon='IcArrowLeftBold' />
+                <Text size='xs' weight='bold' color='prominent'>
+                    <Localize i18n_default_text="Trader's hub" />
+                </Text>
+            </div>
+            <h1 className='compare-cfd-header-title'>
+                <Text size='m' weight='bold' color='prominent'>
+                    <Localize
+                        i18n_default_text='Compare CFDs {{demo_title}} accounts'
+                        values={{
+                            demo_title: is_demo ? localize('demo') : '',
+                        }}
+                    />
+                </Text>
+            </h1>
+        </div>
+    );
+
+    return (
+        <React.Fragment>
+            <DesktopWrapper>
+                <div className='compare-cfd-account'>
+                    <PageOverlay header={DesktopHeader} is_from_app={routes.traders_hub} />
+                    <div
+                        className={classNames('compare-cfd-account-container', {
+                            'compare-cfd-account-container__card-count': card_count < 4,
+                        })}
+                    >
+                        <div className='card-list'>
+                            <CFDCompareAccountsCarousel>
+                                {all_cfd_available_accounts.map(item => (
+                                    <CFDCompareAccountsCard
+                                        trading_platforms={item}
+                                        key={item.market_type + item.shortcode}
+                                        is_eu_user={is_eu_user}
+                                        is_demo={is_demo}
+                                    />
+                                ))}
+                                {/* Renders Deriv X data */}
+                                {all_cfd_available_accounts.length > 0 && has_dxtrade_account_available && (
+                                    <CFDCompareAccountsCard
+                                        trading_platforms={dxtrade_data}
+                                        is_eu_user={is_eu_user}
+                                        is_demo={is_demo}
+                                    />
+                                )}
+                            </CFDCompareAccountsCarousel>
+                        </div>
+                    </div>
+                </div>
+            </DesktopWrapper>
+            <MobileWrapper>
+                <PageOverlay
+                    header={
+                        <Localize
+                            i18n_default_text='Compare CFDs {{demo_title}} accounts'
+                            values={{
+                                demo_title: is_demo ? localize('demo') : '',
+                            }}
```

Comment:
same here

Reply:
Nope i already checked in real(is_demo=false) it is "Compare CFDs accounts"